pkgs/applications/misc/huggle/01-extensions.patch

```diff
@@ -0,0 +1,112 @@
+index 05fec89..6f61888 100644
+--- a/src/extensions/extension-mass-delete/CMakeLists.txt
++++ b/src/extensions/extension-mass-delete/CMakeLists.txt
+@@ -56,5 +56,5 @@ INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
+ if (LINUX_SNAP)
+     INSTALL(TARGETS huggle_nuke DESTINATION bin/extensions)
+ else()
+-    INSTALL(TARGETS huggle_nuke DESTINATION share/huggle/extensions)
++    INSTALL(TARGETS huggle_nuke DESTINATION @out@/extensions)
+ endif()
+Submodule src/extensions/extension-scoring contains modified content
+diff --git a/src/extensions/extension-scoring/CMakeLists.txt b/src/extensions/extension-scoring/CMakeLists.txt
+index f459515..b97ada6 100644
+--- a/src/extensions/extension-scoring/CMakeLists.txt
++++ b/src/extensions/extension-scoring/CMakeLists.txt
+@@ -76,5 +76,5 @@ INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
+ if (LINUX_SNAP)
+     INSTALL(TARGETS huggle_scoring DESTINATION bin/extensions)
+ else()
+-    INSTALL(TARGETS huggle_scoring DESTINATION share/huggle/extensions)
++    INSTALL(TARGETS huggle_scoring DESTINATION @out@/extensions)
+ endif()
+Submodule src/extensions/extension-splitter-helper contains modified content
+diff --git a/src/extensions/extension-splitter-helper/CMakeLists.txt b/src/extensions/extension-splitter-helper/CMakeLists.txt
+index d56f8a5..46ede3e 100644
+--- a/src/extensions/extension-splitter-helper/CMakeLists.txt
++++ b/src/extensions/extension-splitter-helper/CMakeLists.txt
+@@ -67,5 +67,5 @@ INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
+ if (LINUX_SNAP)
+     INSTALL(TARGETS huggle_sh DESTINATION bin/extensions)
+ else()
+-    INSTALL(TARGETS huggle_sh DESTINATION share/huggle/extensions)
++    INSTALL(TARGETS huggle_sh DESTINATION @out@/extensions)
+ endif()
+Submodule src/extensions/extension-thanks contains modified content
+diff --git a/src/extensions/extension-thanks/CMakeLists.txt b/src/extensions/extension-thanks/CMakeLists.txt
+index f4bfa1b..eea28eb 100644
+--- a/src/extensions/extension-thanks/CMakeLists.txt
++++ b/src/extensions/extension-thanks/CMakeLists.txt
+@@ -67,6 +67,6 @@ INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
+ if (LINUX_SNAP)
+     INSTALL(TARGETS huggle_thanks DESTINATION bin/extensions)
+ else()
+-    INSTALL(TARGETS huggle_thanks DESTINATION share/huggle/extensions)
++    INSTALL(TARGETS huggle_thanks DESTINATION @out@/extensions)
+ endif()
+ 
+Submodule src/extensions/mass-delivery contains modified content
+diff --git a/src/extensions/mass-delivery/CMakeLists.txt b/src/extensions/mass-delivery/CMakeLists.txt
+index 29c52fa..189b06d 100644
+--- a/src/extensions/mass-delivery/CMakeLists.txt
++++ b/src/extensions/mass-delivery/CMakeLists.txt
+@@ -75,5 +75,5 @@ INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
+ if (LINUX_SNAP)
+     INSTALL(TARGETS huggle_md DESTINATION bin/extensions)
+ else()
+-    INSTALL(TARGETS huggle_md DESTINATION share/huggle/extensions)
++    INSTALL(TARGETS huggle_md DESTINATION @out@/extensions)
```

Comment:
This path seems reasonable to me and already in $out. Why change it? Maybe we just need to change DESTINATION? We should try to do as little patching as possible.